connexion/apis/flask_api.py

```diff
@@ -300,9 +312,16 @@ def console_ui_home(self):
 
         :return:
         """
+        openapi_json_route_name = "{blueprint}.{prefix}_openapi_json"
```

Comment:
In line 44 and here we do similar formatting for the register and to get the URL for the endpoint. Maybe we can do it once? I would like to avoid duplicated "..._openapi_json" string. Something could tie this two together closer, so we don't break things later. I know we have tests covering this, but if possible would be nice to remove this duplication. Sorry if this seems over-engineering.

Reply:
Hey @rafaelcaricio  - really appreciate your comments.
I agree with your point here, but I couldn't come up with a good idea for how to make it happen. Maybe you could provide a more specific example of what you have in mind?